docs/pages/access-controls/guides/device-trust.mdx

```diff
@@ -0,0 +1,326 @@
+---
+title: "Device Trust (Preview)"
+description: Learn how to use and enforce trusted devices with Teleport
+---
+
+<Admonition type="warning" title="Preview">
+  Device Trust is currently in Preview mode.
+</Admonition>
+
+<ScopedBlock scope="oss">
+  Device Trust requires a commercial edition of Teleport.
+</ScopedBlock>
+
+Device Trust allows Teleport admins to enforce the use of trusted devices.
+Resources protected by the device mode "required" will enforce the use of a
+trusted device, in addition to establishing the user's identity and enforcing
+the necessary roles. Furthermore, users using a trusted device leave audit
+trails that include the device's information.
+
+The device trust preview supports the following Teleport features:
+
+- SSH access enforcement
+- Database access enforcement
+- Kubernetes access enforcement
+- macOS trusted devices
+
+Support for other operating systems and access features is planned for upcoming
+Teleport versions.
+
+## Concepts
```

Comment:
Some of the feedback we've seen from the community lately is a desire to see more "how it works" docs instead of framing everything as a guide.

I wonder if it makes sense to pull some of these concepts out into a separate intro page for people who want to understand the feature but don't care (or aren't ready) to read through a step-by-step guide on how to set it up.

Reply:
I thought briefly about a separate page too. Maybe we could move this part under architecture?

The reason I added it here is that, unlike other features, device trust isn't self-explanatory. Even a weakly-opinionated system like our has to make some calls on how devices are registered, enrolled, etc, so it felt necessary to explain a bit of how it works here. I'm open to a refactor, though.

Reply:
I think the volume of explanation here is small enough to warrant a single page. Any more, and we'd want to separate it.